src/dynamicprompts/wildcards/wildcard_manager.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import annotations
+
+import logging
+from fnmatch import fnmatch
+from pathlib import Path
+from typing import Iterable
+
+from dynamicprompts.parser.config import default_parser_config
+from dynamicprompts.wildcards.utils import clean_wildcard
+from dynamicprompts.wildcards.wildcard_file import WildcardFile
+from dynamicprompts.wildcards.wildcard_tree import (
+    WildcardTreeNode,
+    build_tree_from_path,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class WildcardManager:
+    def __init__(
+        self,
+        path: Path | None = None,
```

Comment:
What are your thoughts on allowing a string path as well as a pathlib Path?

Reply:
Sure, that makes sense. It'd be just one cast within the ctor anyway. :)